feat: turn off notebook service

olevski opened the pull request:
This turns off the notebook service and uses the data service to manage JupyterServers.

/deploy

I tested in another throwaway CI deployment that when we update the sessions from v1 and v2 sessions are still visible to users. For both registered and anonymous users.

Panaetius commented on helm-chart/values.yaml.changelog.md:
```diff
@@ -7,6 +7,10 @@ Please follow this convention when adding a new row
 
 ## Upgrading to Renku 0.62.0
 
+* EDIT ``notebooks.replicaCount`` - set to zero by default because the data service is handling all sessions now, the notebook service will be fully decomissioned and remove from the helm chart in a subsequent PR.
```
```suggestion
* EDIT ``notebooks.replicaCount`` - set to zero by default because the data service is handling all sessions now, the notebook service will be fully decomissioned and removed from the helm chart in a subsequent PR.
```

aledegano commented on helm-chart/values.yaml.changelog.md:
```diff
@@ -7,6 +7,10 @@ Please follow this convention when adding a new row
 
 ## Upgrading to Renku 0.62.0
```
```suggestion
## Upgrading to Renku 0.66.0
```

olevski replied:
Omg. Ok I will fix this. Also will wait for the custom envs build PR to be merged then rebase and merge this.

bethcg reviewed (approved):
Just add the .  at the end of each of the improvements in CHANGELOG.rst to be consistent with the rest of the document.

olevski commented:
> Just add the . at the end of each of the improvements in CHANGELOG.rst to be consistent with the rest of the document.

I will open a separate PR for this.

lorenzo-cavazzi commented:
I'm not sure this is working fine. I can start v1 sessions but the UI doesn't show any and I can't connect.
This is the same error I'm getting in this renku-notebooks PR, so perhaps the latest version of renku-notebooks isn't working. Or values are missing in the PR deployments? In that case, you could try adding them in the `/deploy` string

olevski commented:
> I'm not sure this is working fine. I can start v1 sessions but the UI doesn't show any and I can't connect.
This is the same error I'm getting in this renku-notebooks PR, so perhaps the latest version of renku-notebooks isn't working. Or values are missing in the PR deployments? In that case, you could try adding them in the /deploy string

@lorenzo-cavazzi I just deleted your session with kubectl. I had a lot of leftover sessions from the acceptance tests so I cleaned everything up. I saw your session but did not see how old it was. And I deleted everything.
